developers/weaviate/quickstart/index.md

```diff
@@ -257,22 +257,22 @@ Generative search sends retrieved data from Weaviate to a large language model,
 ## Recap
 
 Well done! You have:
-- Created your own cloud-based vector database with Weaviate,
-- Populated it with data objects using an inference API,
+- Created your own cloud-based vector database with Weaviate
+- Populated it with data objects using an inference API
 - Performed searches, including:
-    - Semantic search,
-    - Semantic search with a filter and
-    - Generative search.
+    - Semantic search
+    - Semantic search with a filter
+    - Generative search
 
 Where next is up to you. We include a few links below - or you can check out the sidebar.
 
 <hr/>
 
 ## Next
 
-You can do much more with Weaviate. We suggest trying:
+You can do much more with Weaviate. We suggest trying any of the below:
 
-- Examples from our [search how-to](../search/index.md) guides for [keyword](../search/bm25.md), [similarity](../search/similarity.md), [hybrid](../search/hybrid.md), [generative](../search/generative.md) searches and [filters](../search/filters.md) or
+- Examples from our [search how-to](../search/index.md) guides for [keyword](../search/bm25.md), [similarity](../search/similarity.md), [hybrid](../search/hybrid.md), [generative](../search/generative.md) searches and [filters](../search/filters.md).
```

Comment:
Consider nesting these something like this:

```shell
- Search examples from our [guides](/developers/weaviate/search)
   -  [Keyword search](link)
   - [Vector search](link)
 - Data management
   - [Reads](link)
   - [Updates](link)
 ```

Reply:
Thanks for your suggestions so far, @mungitoperrito. I believe nesting isn’t important here because when a user clicks on “search how-to”, they’ll see a list of the search syntaxes.